rules/preprocessing.smk

```diff
@@ -65,9 +59,9 @@ rule annotate:
 
 rule normalize:
     input:
-        expand("results/preprocessing/{plate_id}_augmented.csv.gz", plate_id=PLATE_IDS),
+        "results/preprocessing/{plate_id}_augmented.csv.gz",
     output:
-        expand("results/preprocessing/{plate_id}_normalized.csv.gz", plate_id=PLATE_IDS),
+        "results/preprocessing/{plate_id}_normalized.csv.gz",
```

Comment:
Do you need to add a consensus rule in this .smk file?

Reply:
The consensus rule is in the `rulesfeature_select.smk` not `preprocessing.smk`. Unless you want the `feature_select` and `consensus` rule in one `.smk` file. 

I guess my question is - Is `feature_select` and `consensus` a preprocessing step?

Reply:
this is a good question, and the answer isn't immediately obvious (to me at least). I'm not sure there even is one!

If I had to guess, I'd say _none_ of this is preprocessing. It is all _processing_.

I think of preprocessing as steps immediately following data extraction from images (illum correction, quality control, the things Jenna has been focused on). I'm not sure where we draw the line between preprocessing and processing, but I think it comes before aggregation even.

Either way, I think separating rules between `rulesfeature_select.smk` and `preprocessing.smk` is a bit confusing (for the reasons I outlined above about how to think about processing vs. preprocessing).

But, I think that fixing this might be beyond scope of this PR - I opened #12 for us to discuss further, and to document that we still need to address this once we merge this present PR.